include/onnxruntime/core/graph/indexed_sub_graph.h

```diff
@@ -30,6 +30,14 @@ and the meta definition needed for representing this subgraph as a FunctionProto
 which could be serialized/saved to a model file.
 */
 struct IndexedSubGraph {
+  enum class SourceOfSchema : uint8_t {
+    CREATE,           /// create new schema from info in IndexedSubGraph instance.
```

Comment:
nit: could we move this enum definition down to immediately prior to the schema_source member? it's not the most important info about this struct so there's no need for it to be first.